app/src/main/java/com/example/android/architecture/blueprints/todoapp/statistics/StatisticsViewModel.kt

```diff
@@ -53,7 +51,7 @@ class StatisticsViewModel @Inject constructor(
     val uiState: StateFlow<StatisticsUiState> =
         tasksRepository.getTasksStream()
             .map { Async.Success(it) }
-            .onStart<Async<Result<List<Task>>>> { emit(Async.Loading) }
+            .onStart<Async<List<Task>>> { emit(Async.Loading) }
```

Comment:
Do we want to start with `Async.Loading` each time the stream is resubscribed to? It will case the same behavior from NiA of losing scroll state.